### DISCUSSION TITLE
Button vs Anchor

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1680] [Q&A] [answered]
An engineer brought up a question around the use of a button on this [page](https://www.figma.com/proto/WTlMlYU1gEcCceHmdviNsQ/Secure-Onboarding?page-id=647%3A620&node-id=693%3A43117&viewport=-842%2C5669%2C0.4063906669616699&scaling=min-zoom) 

The question was around whether the "Continue to Setup Guide" should be a button or an anchor?

**Comment by TheSisb** [comment, marked as answer]
Hi @wilfredomerced,

If that button is performing an in-page action, then a Button is fine.  If it is redirecting to a new page, then the preferred approach for accessibility is to use an Anchor. 

We do have [Buttons that behave like Anchors](https://paste.twilio.design/patterns/button-vs-anchor#buttons-with-link-functionality) as seems to be used in the mockup. Please only use those as a last resort as it is not as accessible of a solution.
